### DISCUSSION TITLE
Extracting masks as ROIs

### DISCUSSION THREAD

**Discussion by dimethylglymaxim** [SchmollerLab/Cell_ACDC#208] [Q&A] [answered]
Hi

I'm interested if there is any way I can extract masks and/or tracking/annotations in a way, that ImageJ can recognize. I can't quite figure out the way to transform .npz to .npy to .roi.

If you have any advice, I would gladly accept it.
Thanks

**Comment by ElpadoCan** [comment, marked as answer]
Hi @dimethylglymaxim ,

There is a utility in Cell-ACDC to achieve exactly that. You can find this utility from the menu on the top menu bar of the little launcher window called `Utilities --> Convert file formats --> Convert _segm.npz to ImgeJ ROIs` (see screenshot attached).

Launch the utility and go through the wizard. Hopefully it is self-explanatory, but if not, feel free to ask how to use it. 

Let me know if it works, thanks! 

Cheers,
Francesco

![image](https://user-images.githubusercontent.com/55240925/203984783-2dcfb8d2-3701-47a9-ae86-ae0e2173530d.png)

**Comment by dimethylglymaxim** [comment]
Absolutely perfect. Thank you very much.
